src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+$secondary: #616070;
+$btn-white-color: #fff;
+$btn-turquoise-color: #00acdc;
+$main-acent: #060b35;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
```

Comment:
The &photo should be &__photo to follow BEM naming conventions, which state that elements of a block should be named using a double underscore.